go.mod

```diff
@@ -7,8 +7,10 @@ require (
 	github.com/container-storage-interface/spec v1.6.0
 	github.com/go-logr/logr v1.2.3
 	github.com/kubernetes-csi/csi-lib-utils v0.11.0
+	github.com/onsi/ginkgo/v2 v2.1.6
+	github.com/onsi/gomega v1.20.1
```

Comment:
It's a shame that we have to pull in these test dependencies in our main go.mod file... Not sure what the best strategies are for avoiding this, but it could well make things harder in future with consumers of this library that e.g. depend on `controller-runtime` to provide a Client (or even depend on cert-manager which itself depends on these libraries)

Reply:
I've gone ahead and created a `go.mod` under `/test/e2e`. This seems to be working quite well.